root/static/scripts/account/components/PreferencesForm.js

```diff
@@ -62,12 +62,11 @@ const allowedDateTimeFormats = [
   '%m.%d.%Y %H:%M',
 ];
 
-function buildDateTimeFormatOptions(timezone) {
-  const hereAndNow = moment.tz(timezone);
+function buildDateTimeFormatOptions($c, hereAndNow, timezone) {
```

Comment:
Is there a reason why we can't calculate hereAndNow inside the function?

Reply:
Don't think so - can't remember why I moved that, so reverted.